core/src/main/java/jenkins/security/DefaultConfidentialStore.java

```diff
@@ -151,5 +152,5 @@ SecureRandom secureRandom() {
         return random;
     }
 
-    private static final byte[] MAGIC = "::::MAGIC::::".getBytes();
+    private static final byte[] MAGIC = "::::MAGIC::::".getBytes(Charset.defaultCharset());
```

Comment:
ASCII or UTF-8

Reply:
`"::::MAGIC::::"` is valid ASCII, so let's use that.